plugins/PagePerformance/Archiver.php

```diff
@@ -54,16 +55,19 @@ public function aggregateDayReport()
             $column = $dimension->getColumnName();
             $selects[] = "sum($table.$column) as {$column}_total";
             $selects[] = "sum(if($table.$column is null, 0, 1)) as {$column}_hits";
-            $totalColumns[] = "$table.$column";
+            $totalColumns[] = "IFNULL($table.$column,0)";
             $hitsColumns[] = "if($table.$column is null, 0, 1)";
+            $allColumns[]  = $table.$column;
```

Comment:
That can't work, as the `.` is used for concatenation, but is required in sql.
```suggestion
            $allColumns[]  = "$table.$column";
```